lib/edit-site-page.php

```diff
@@ -160,12 +160,8 @@ function gutenberg_edit_site_init( $hook ) {
 		}
 	}
 
-	$current_template_id = $template_ids['front-page'];
-
 	$settings['editSiteInitialState'] = array();
 
-	$settings['editSiteInitialState']['homeTemplateId']  = $current_template_id;
-	$settings['editSiteInitialState']['templateId']      = $current_template_id;
 	$settings['editSiteInitialState']['templateType']    = 'wp_template';
```

Comment:
I guess we can also get rid of all of these except for `showOnFront` unless that is already in the site entity.

Reply:
Cool. I'll look into removing these in a follow-up :)